ui/pages/settings/networks-tab/networks-form/networks-form.js

```diff
@@ -130,6 +130,7 @@ const NetworksForm = ({
   const DEFAULT_SUGGESTED_NAME = [];
   const CHAIN_LIST_URL = 'https://chainid.network/';
   const { label, labelKey, viewOnly, rpcPrefs } = selectedNetwork;
+
```

Comment:
We need to be careful here with any changes as they would effect the non-feature-flagged MetaMask